### ISSUE TITLE
Ballet studios / dancing schools

### ISSUE THREAD

**Issue by FloEdelmann** [openstreetmap/id-tagging-schema#1177]
I searched for "Ballettstudio" (in German) and nothing was found. In a [forum thread](https://community.openstreetmap.org/t/ballett-schule/86169) I found that I should rather search for dancing school ("Tanzschule" in German) and add ballet as a dancing style there. This worked. So **first suggestion:** Can we add "ballet studio" as a term for the Dancing School preset?

However, the Dancing School preset (`amenity = dancing_school`) was marked as deprecated and suggested replacing it with the Dance Hall preset preset (`leisure = dance` + `dance:teaching = yes`), which was fine for me. It seems wrong to have a suggested preset that is marked as deprecated though. So **second suggestion:** Should we remove the Dancing School preset and add its terms to the Dance Hall preset?

Note also that there is an issue about a wrong deprecation of `amenity = dancing_school`:

* https://github.com/openstreetmap/id-tagging-schema/issues/373

So maybe the deprecation is wrong at all?

**Comment by simonpoole** [comment]
This is a leftover of the previous reign of maintainers were the policy was "we tag everything in our myopic view and basta".  There is unluckily still a lot of legacy from that period left (for example office=coworking).

**Comment by bhousel** [comment]
> This is a leftover of the previous reign of maintainers were the policy was "we tag everything in our myopic view and basta". There is unluckily still a lot of legacy from that period left (for example office=coworking).

It's really uncool of you to write stuff like this.   Be better, Simon.

For the curious, here's a link to the thing you are still mad about:  https://github.com/openstreetmap/iD/pull/6251

TLDR - 5 years ago, @westnordost sent me a PR to deprecate a thing based on what the wiki said, and I said "sure".   Somehow that makes me a tag tyrant in Simon's view.

**Comment by simonpoole** [comment]
That was not about you falling indirectly for a notorious wiki fiddler, but you refusing to fix the error once it was pointed out to you.

**Comment by tyrasd** [comment]
@FloEdelmann 

> second suggestion

Yeah, there should never be a preset for a tag which is also in the "deprecated" tags list.

However, our intention was to have both the Dancing School and Dance Hall presets to be valid, proper presets of related, but slightly different map features. I had thought that this was already fixed with #814, but it looks like we had accidentally change the wrong file (the `dist` folder is overwritten by the build scripts, so that PR had little effect) :see_no_evil: 915ed76ac030533c058b8cc84f36ff99616c646f implements this now properly. Sorry for the oversight :bow: and thanks for reporting this discrepancy!

**Comment by tyrasd** [comment]
> I searched for "Ballettstudio" (in German) and nothing was found.

This is most likely just missing in the German translations of the search terms of this preset. However, translations are not managed on github, but on a dedicated platform called transifex (see https://github.com/openstreetmap/id-tagging-schema/blob/main/CONTRIBUTING.md#Translating). You should be able to fix the German strings on there.
